pom.xml

```diff
@@ -73,6 +72,11 @@
 			<artifactId>spring-security-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
```

Comment:
remove dependencies

Reply:
This can be done with the git command 
git checkout origin/dev -- pom.xml